modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/scim2/SCIM2UserTestCase.java

```diff
@@ -287,6 +296,75 @@ public void testGetResourceTypes() throws Exception {
 
     }
 
+    @Test
+    public void testUpdateUserWhenExternalClaimDeleted() throws Exception {
+
+        AutomationContext context = new AutomationContext("IDENTITY", testUserMode);
+        backendURL = context.getContextUrls().getBackEndUrl();
+        loginLogoutClient = new LoginLogoutClient(context);
+        sessionCookie = loginLogoutClient.login();
+        HttpPost postRequest = new HttpPost(getPath());
+        postRequest.addHeader(HttpHeaders.AUTHORIZATION, getAuthzHeader());
+        postRequest.addHeader(HttpHeaders.CONTENT_TYPE, "application/json");
+
+        JSONObject rootObject = new JSONObject();
+
+        JSONArray schemas = new JSONArray();
+        rootObject.put(SCHEMAS_ATTRIBUTE, schemas);
+
+        JSONObject names = new JSONObject();
+        names.put(FAMILY_NAME_ATTRIBUTE, "udaranga");
+        names.put(GIVEN_NAME_ATTRIBUTE, "buddhima");
+
+        rootObject.put(NAME_ATTRIBUTE, names);
+        rootObject.put(USER_NAME_ATTRIBUTE, "wso2is");
+
+        JSONObject emailWork = new JSONObject();
+        emailWork.put(TYPE_PARAM, EMAIL_TYPE_WORK_ATTRIBUTE);
+        emailWork.put(VALUE_PARAM, EMAIL_TYPE_WORK_CLAIM_VALUE);
+
+        JSONObject emailHome = new JSONObject();
+        emailHome.put(TYPE_PARAM, EMAIL_TYPE_HOME_ATTRIBUTE);
+        emailHome.put(VALUE_PARAM, EMAIL_TYPE_HOME_CLAIM_VALUE);
+
+        JSONArray emails = new JSONArray();
+        emails.add(emailWork);
+        emails.add(emailHome);
+
+        rootObject.put(EMAILS_ATTRIBUTE, emails);
+
+        rootObject.put(PASSWORD_ATTRIBUTE, PASSWORD);
+
+        StringEntity entity = new StringEntity(rootObject.toString());
+        postRequest.setEntity(entity);
+        HttpResponse postResponse = client.execute(postRequest);
+        assertEquals(postResponse.getStatusLine().getStatusCode(), 201,
+                "User " + "has not been created in patch process successfully");
```

Comment:
Shall we make the line as one sentence?

Reply:
Fixed https://github.com/wso2/product-is/pull/7360